frontend/packages/ui/src/components/menu-options/menu-options.module.css

```diff
@@ -12,10 +12,28 @@
   gap: 12px;
 }
 
-.user-svg {
+.userSvg {
   margin-left: 2px;
 }
 
-.user-svg path {
+.svg path {
   fill: #000000
+}
+
+.disconnectButton.disconnectButton {
```

Comment:
```suggestion
.disconnectButton {
```
is it typo?

Reply:
it's styles specificity increasing. Styles of button can overlap custom ones sometimes